pom.xml

```diff
@@ -190,8 +190,8 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
-        <ballerina.version>1.2.4</ballerina.version>
-        <module.ftp.version>0.5.1</module.ftp.version>
+        <ballerina.version>2.0.0-SNAPSHOT</ballerina.version>
```

Comment:
We need to change the ballerina version to the released one

Reply:
Corrected.